src/Item/ViewItem/ItemAcquisition/ItemAcquisition.test.js

```diff
@@ -75,9 +75,61 @@ describe('ItemAcquisition', () => {
     expect(screen.getByText(`ui-inventory.acq.receiptStatus.${orderLine.receiptStatus}`)).toBeInTheDocument();
   });
 
-  it('should display fetched item acquisition data - vendor code', () => {
+  it('should display fetched item acquisition data - vendor code with link', () => {
     renderItemAcquisition({ itemId: 'itemId' });
+    const link = screen.getByRole('link', { name: 'AMAZ' });
 
     expect(screen.getByText(vendor.code)).toBeInTheDocument();
+    expect(link).toBeInTheDocument();
+    expect(link).toHaveAttribute('href', '/organizations/view/vendorId');
+  });
+
+  it('should display fetched item acquisition data - vendor name with link', () => {
+    renderItemAcquisition({ itemId: 'itemId' });
+    const link = screen.getByRole('link', { name: 'Amazon' });
+
+    expect(screen.getByText(vendor.name)).toBeInTheDocument();
+    expect(link).toBeInTheDocument();
+    expect(link).toHaveAttribute('href', '/organizations/view/vendorId');
+  });
+
+  it('should display fetched item acquisition data - acquisition method', () => {
+    renderItemAcquisition({ itemId: 'itemId' });
+
+    expect(screen.getByText(orderSetting.value)).toBeInTheDocument();
+  });
+
+  it('should display fetched item acquisition data - calculation and formatting of average cost', () => {
+    renderItemAcquisition({ itemId: 'itemId' });
+
+    const totalAmountExpended = finance.transactions.reduce((sum, transaction) => sum + transaction.encumbrance.amountExpended, 0);
```

Comment:
Please avoid business logic in unit tests - instead just check for a hardcoded expected value